### PR TITLE
[fix] Share GreenBlock link

### PR DESCRIPTION
**Pull request by amir1995**
[add] After submit tree or update tree (navigate to my trees/my communities)
[fix] Pull to refresh not works in profile
[add] Need help? text in profile page
[improve] tree style on my tree list
[increase] limit number for getting tree list

### CODE REVIEW

File: src/screens/GreenBlock/GreenBlock.tsx [hunk 1]
```diff
@@ -27,6 +27,18 @@ function GreenBlock({navigation, route}: Props) {
   const greenBlockIdToJoin = route.params?.greenBlockIdToJoin;
   const alertPending = useRef(false);
 
+  const shouldNavigateToTree = useCallback(() => {
+    if (route.params && route.params.goTree) {
```
Comment (mohebifar) [inline src/screens/GreenBlock/GreenBlock.tsx:31]: You can use optional chaining like so `route.params?.goTree`

File: src/screens/MainTabs.tsx [hunk 2]
```diff
@@ -44,6 +44,7 @@ function MainTabs({navigation}: Props) {
         const [, greenBlockIdToJoin] = matches;
         navigation.navigate('GreenBlock', {
           greenBlockIdToJoin,
+          goTree: false,
```
Comment (mohebifar) [inline src/screens/MainTabs.tsx:47]: It's not really clear by its name what this param is for. Can we change this to `shouldNavigateToTreeDetails`?

**Review by mohebifar — APPROVED** [review]
Looks great! Just a few comments.

File: src/screens/MainTabs.tsx (hunk 2 above)
Comment (amir1995) [reply src/screens/MainTabs.tsx:47]: yes we can.

File: src/screens/GreenBlock/GreenBlock.tsx (hunk 1 above)
Comment (amir1995) [reply src/screens/GreenBlock/GreenBlock.tsx:31]: I got it.

**Pull request closed without merging**